LayoutTests/fast/table/inline-table-margin-baseline-expected.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+.container {
+    border: solid 1px black;
+}
+.vertical {
+  writing-mode: vertical-rl
+}
+.block {
+    display:inline-block;
+    background: aqua;
+}
+.table {
+    display: inline-table;
+    font-size: 40px;
+    background: tan;
+}
+.h-margin {
+    margin-top: 50px;
+}
+.v-margin {
+    margin-right: 50px;
+}
```

Comment:
Nit: unless you think it's important to use physical properties, you could use `margin-block-start: 50px` in `.table` and remove `.h-table` and `.v-table`.

Reply:
Used logical properties